src/granite/exceptions.cr

```diff
@@ -0,0 +1,13 @@
+module Granite
+  class RecordInvalid < ::Exception
```

Comment:
Would this be more grammatically correct as "InvalidRecord"?

Reply:
I followed the Rails convention you mentioned in the comment. So i introduced RecordNotDestroyed and RecordInvalid into the stack. Do you want me to change it to InvalidRecord instead?

Reply:
Generally with Amber I'm not as interested in following in Rails' footsteps as I am in producing something that feels natural. Sometimes that means features similar to Rails, sometimes not.